pkg/operator/mon/mon.go

```diff
@@ -41,6 +41,10 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mon")
 
 const (
+	// EndpointConfigMapName is the name of the configmap with mon endpoints
+	EndpointConfigMapName = "rook-ceph-mon-endpoints"
+	// Endpointkey is the name of the key inside the mon configmap to get the endpoints
+	DataKey           = "data"
```

Comment:
The comment should also match the name of the variable. Endpointkey != DataKey.
I was also thinking that since this is the only variable that needs to have its visibility public, would it be better to create a getter function? It is just odd that all the vars in this struct are private except for this one.